pythonforandroid/bootstraps/service_only/build/src/org/kivy/android/PythonService.java

```diff
@@ -136,7 +131,6 @@ public void onDestroy() {
     @Override
     public void run() {
         PythonUtil.loadLibraries(getFilesDir());
-        mService = this;
```

Comment:
Exactly. Unless you need something to be a singleton, using static variables to hold instances of classes is not what you want.